README.md

```diff
@@ -61,6 +61,33 @@ To launch the complete test matrix run:
     $ tox
 
 
+#### Running Tests in docker
+
+If you prefer not to setup your local machine to run tests, we provide a preconfigured docker image.
+Note that this image is the same used in CircleCI to run tests.
+
+You still need docker containers running additional services up and running.
+
+Run the test runner
+
+    $ docker-compose run --rm testrunner
+
+Now you are in a bash shell. You can now run tests as you would do in your local environment:
+
+    $ tox -e '{py35,py36}-redis{210}'
+
+If you are in a unix machine, we also provide a shell script to execute commands in the provided container (so you don't
```

Comment:
> If you are using a unix machine, we also provide a helper script for running commands directly in this container.

?

Reply:
It is a shell script, so I guess it is not running on Windows? (not a Windows user). I will replace with `MAC or Linux machine`: ack?